src/lib-zephyr/cpu.c

```diff
@@ -123,6 +123,7 @@ int cpu_enabled_cores(void)
 
 #else /* CONFIG_ACE */
 #include <sof/trace/trace.h>
+#include <rtos/wait.h>
```

Comment:
The change is evident, but the commit leaves me wondering in which configuration? We have continuous build tests for ACE in SOF main, so in which configuration the build fails? 

If this is just to make code future-proof, better to reflect this in commit message.

Reply:
@kv2019i This could be needed as we de-tangle the rtos and generic headers. This will be ongoing and should be able to be completed after full native Zephyr mode is working upstream.